tests/integration/test_s3.py

```diff
@@ -555,7 +555,7 @@ def test_s3_presigned_post_success_action_status_201_response(self):
                     <ETag>{etag}</ETag>
                 </PostResponse>
                 """.format(
-            location='http://localhost/key-my-file',
+            location='http://localhost:4566/' + bucket_name + '/key-my-file',
```

Comment:
Can we extend this test to use an object name that contains a slash (`/`)? 

I'm not entirely sure why we're using `urlparse.quote` with `safe=''` above, as the following code generates a key with encoded slashes:
```
> urlparse.quote('foo/bar', safe=''),
('foo%2Fbar',)
```
My feeling is that we may have to change the code above to something like:
```
return '{protocol}://{host}/{bucket}/{encoded_key}'.format(
...
encoded_key=urlparse.quote(key),
...
```

Reply:
Hi @rhertogh , quick follow up - do you think we could slightly extend/change the test to include a slash in the object key? I'm not entirely sure if the functionality would be affected,  but it seems like the key name would get formatted as `foo%2Fbar`, right..? Thanks for your help!